apiary.apib

```diff
@@ -2029,6 +2029,8 @@ Lists buckets which may be shared to the project.
 
 ### Create new table from CSV file [POST]
 
+***DEPRECATED**, use [Create table asynchronously](#reference/tables/create-table-asynchronously/create-new-table-from-csv-file-asynchronously) instead.*
```

Comment:
Je to deprecated, ne? 🙂

Reply:
Měl by ho nahradit ten [create table definition](https://keboola.docs.apiary.io/#reference/tables/create-or-list-tables/create-new-table-definition) ale ten je ještě přes feature tak nevím jeslti se tenhle api call už může označit za deprecated. To je asi na @zajca

Reply:
to je OK, on píše že je deprecad sync akce a má sa použit async

Reply:
aha, sorry to jsem spatne cetl, tak to je ok